app/i18n/locales/en.json

```diff
@@ -1,776 +1,777 @@
-	"Last_owner_team_room": "You are the last owner of this channel. Once you leave the team, the channel will be kept inside the team but you will be managing it from outside.",
-	"last-owner-can-not-be-removed": "Last owner cannot be removed",
-	"Remove_User_Teams": "Select channels you want the user to be removed from.",
-	"Delete_Team": "Delete Team",
-	"Select_channels_to_delete": "This can't be undone. Once you delete a team, all chat content and configuration will be deleted. \n\nSelect the channels you would like to delete. The ones you decide to keep will be available on your workspace. Notice that public channels will still be public and visible to everyone.",
-	"You_are_deleting_the_team": "You are deleting this team.",
-	"Removing_user_from_this_team": "You are removing {{user}} from this team",
-	"Remove_User_Team_Channels": "Select the channels you want the user to be removed from.",
-	"Remove_Member": "Remove Member",
-	"leaving_team": "leaving team",
-	"removing_team": "removing from team",
-	"moving_channel_to_team": "moving channel to team",
-	"deleting_team": "deleting team",
-	"member-does-not-exist": "Member does not exist",
-	"Convert": "Convert",
-	"Convert_to_Team": "Convert to Team",
-	"Convert_to_Team_Warning": "You are converting this Channel to a Team. All Members will be kept.",
-	"Move_to_Team": "Move to Team",
-	"Move_Channel_Paragraph": "Moving a channel inside a team means that this channel will be added in the team’s context, however, all channel’s members, which are not members of the respective team, will still have access to this channel, but will not be added as team’s members. \n\nAll channel’s management will still be made by the owners of this channel.\n\nTeam’s members and even team’s owners, if not a member of this channel, can not have access to the channel’s content. \n\nPlease notice that the Team’s owner will be able remove members from the Channel.",
-	"Move_to_Team_Warning": "After reading the previous intructions about this behavior, do you still want to move this channel to the selected team?",
-	"Load_More": "Load More",
-	"Load_Newer": "Load Newer",
-	"Load_Older": "Load Older",
-	"room-name-already-exists": "Room name already exists",
-	"error-team-creation": "Error team creation",
-	"unauthorized": "Unauthorized",
-	"Left_The_Room_Successfully": "Left the room successfully",
-	"Deleted_The_Team_Successfully": "Team deleted successfully",
-	"Deleted_The_Room_Successfully": "Room deleted successfully",
-	"Convert_to_Channel": "Convert to Channel",
-	"Converting_Team_To_Channel": "Converting Team to Channel",
-	"Select_Team_Channels_To_Delete": "Select the Team’s Channels you would like to delete, the ones you do not select will be moved to the Workspace. \n\nNotice that public Channels will be public and visible to everyone.",
-	"You_are_converting_the_team": "You are converting this Team to a Channel",
-	"creating_discussion": "creating discussion"
+  "1_person_reacted": "1 person reacted",
```

Comment:
I didn't get where this is coming from.
Maybe an issue on Prettier?

Reply:
Yep!